workflow-templates/debian-package-deliver-pipeline.properties.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "Debian Deliver Package to Artifactory Pipeline",
+    "description": "This pipeline will take a specific Debian package and deliver it to Artifactory.",
+    "iconName": "octicon package",
+    "categories": [
+        "Artifactory",
+        "Debian"
+    ],
+    "filePatterns": [
+        ".deb"
```

Comment:
Isn't the .deb the output so it's unlikely to be in the repo for github to find to suggest this workflow. Leave it blank if that's the case or replace with some other file that would hint this repo is a .deb build.

Reply:
Or is the build file also called .deb?

Reply:
When you build the package you do something like `dpkg-deb --build package` and it will output a `package.deb` file. Dennis had mentioned during scrum that the build artifacts are actually versioned and uploaded to VCS so I was relying on that knowledge

Reply:
OK if they are checked in then I guess that's ok (Although not good for Git to have Blobs... And if so, why not checkout the git repo any bypass Artifactory entirely...)

If not, this is used at pipeline creation time so would need to be removed if the repo does not have a .deb file or no-one will discover this pipeline from the GitHub add new action screen.